readme.md

```diff
@@ -1,4 +1,7 @@
 # Spring PetClinic Sample Application [![Build Status](https://travis-ci.org/spring-projects/spring-petclinic.png?branch=master)](https://travis-ci.org/spring-projects/spring-petclinic/)
+<a href="https://push-to.cfapps.io?repo=https%3A%2F%2Fgithub.com%2Fspring-projects%2Fspring-petclinic.git">
```

Comment:
Can we get a little blurb there maybe to explain in more details what that means? Also, 200 looks quite big to me.

Reply:
I have made the button "inlined" and smaller... Tried to place it at the beginning of the description string but doesn't seem like GitHub MD understands what i want as it places the text on the next line.
We could place the button on a separate line below if you like...